build/webvr-polyfill.js

```diff
@@ -1954,17 +1953,11 @@ var Viewers = {
     interLensDistance: 0.064,
     baselineLensDistance: 0.035,
     screenLensDistance: 0.039,
-    distortionCoefficients: [0.34, 0.55],
-    inverseCoefficients: [-0.33836704, -0.18162185, 0.862655, -1.2462051,
```

Comment:
It seems `inverseCoefficients` (and `DEFAULT_LEFT(RIGHT)_CENTER`) are no longer used.